src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -13,6 +14,14 @@ const DISMISS_CLASSNAMES = ["color__label", "color__swatch"];
 class ThemeColorsEditor extends React.Component {
   constructor(props) {
     super(props);
+    this.colorPicker = null;
+    this.setColorPickerRef = element => {
+      this.colorPicker = element;
+    };
+    const colors = this.props.theme.colors;
+    this.lastSelectedColors = {
```

Comment:
There is no need for storing the last known colors on all individual themes. The value would be lost anyway when the page refreshes.

There is only a need to remember one color, because otherwise the color picker became jumpy. See my comment below.